release-notes/versioned/pulsar-3.1.0.md

```diff
@@ -0,0 +1,282 @@
+- [MISC] Catch TimeoutException when logging about time outs [#20349](https://github.com/apache/pulsar/pull/20349)
+- [Schema] Only handle exception when there has [#20730](https://github.com/apache/pulsar/pull/20730)
+- [Schema] Add admin cli for testCompatibility [#19974](https://github.com/apache/pulsar/pull/19974)
+- [Build] Fix generating javadoc issue [#20922](https://github.com/apache/pulsar/pull/20922)
+- [build] Do not bundle swagger dependencies [#20687](https://github.com/apache/pulsar/pull/20687)
+- [Build] Build function-go in golang image [#20665](https://github.com/apache/pulsar/pull/20665)
+- [Build] Configure git-commit-id-plugin to skip git describe [#20550](https://github.com/apache/pulsar/pull/20550)
+- [Build] Fix the pulsar-all image may use the wrong upstream image [#20435](https://github.com/apache/pulsar/pull/20435)
+- [Build] Fix publish image script [#20305](https://github.com/apache/pulsar/pull/20305)
+- [Build] Capture local build scans on ge.apache.org to benefit from deep build insights [#20187](https://github.com/apache/pulsar/pull/20187)
+
+### Library updates
+- Upgrade Guava to 32.1.1 to address CVE-2023-2976 [#20699](https://github.com/apache/pulsar/pull/20699)
+- Upgrade snappy-java to address multiple CVEs [#20604](https://github.com/apache/pulsar/pull/20604)
+- Upgrade Jetty to address CVEs [#20461](https://github.com/apache/pulsar/pull/20461)
+- Suppress etcd CVE warnings [#20342](https://github.com/apache/pulsar/pull/20342)
+- Upgrade PyYaml version to 6.0.1 [#20831](https://github.com/apache/pulsar/pull/20831)
+- Upgrade jackson version to 2.15.0 for CVE-2022-1471 [#20177](https://github.com/apache/pulsar/pull/20177)
+- Upgrade prometheus client_golang to v1.12.2 to fix CVE-2022-21698 [#20579](https://github.com/apache/pulsar/pull/20579)
+- Upgrade Netty to 4.1.93.Final [#20423](https://github.com/apache/pulsar/pull/20423)
+- Upgrade to jetcd to 0.7.5 [#20339](https://github.com/apache/pulsar/pull/20339)
+- Upgrade Kotlin version from 1.4.32 to 1.8.20. (targeted for master) [#20114](https://github.com/apache/pulsar/pull/20114)
+
+### Documentation
+- Correcting spelling mistakes in the pulsar-broker module [#20798](https://github.com/apache/pulsar/pull/20798)
+- Fix doc of admin cli create-subscription [#20680](https://github.com/apache/pulsar/pull/20680)
+- Change the method of generating reference docs titles. [#20557](https://github.com/apache/pulsar/pull/20557)
+- Improve doc for delayed message [#20374](https://github.com/apache/pulsar/pull/20374)
+- Correcting spelling mistakes [#20340](https://github.com/apache/pulsar/pull/20340)
+- Correcting spelling mistakes [#20329](https://github.com/apache/pulsar/pull/20329)
+- ConnectorDocGenerator support Java 9+ [#20100](https://github.com/apache/pulsar/pull/20100)
+- Add link to PIP proposal [#19911](https://github.com/apache/pulsar/pull/19911)
+
+Here are the most noteworthy changes in this release.
```

Comment:
Should this sentence be moved to row 263? And the highlights may need an update to include the docs for the new Load Balancer? //cc @Anonymitaet

Reply:
@momo-jun thanks for your reminder, yes, it should include the new Load Balancer doc

Reply:
@Technoboy- noteworthy doc changes should be pulled from the pulsar-site repo rather than the pulsar repo